raiden/utils/runnable.py

```diff
@@ -98,3 +98,8 @@ def on_success(greenlet: Greenlet) -> None:
 
     def __bool__(self) -> bool:
         return bool(self.greenlet)
+
+    def rawlink(self, callback: Callable) -> None:
```

Comment:
Question: When would this be called? couldn't we just replace the call to `x.greenlet.rawlink` or is this internal greenlet stuff?

Reply:
Check the issue. I documented it there: https://github.com/raiden-network/raiden/issues/5327

TL;DR: It's gevent internal inside `joinall`.

Reply:
Cleared. Thanks